config.py

```diff
@@ -79,6 +82,30 @@ def __init__(self):
         execution engine. Must be supplied.
         """
 
+        # ==========================================================================================
+        # Geth Configuration
+
+        self.data_dir = L2_EXECUTION_DATA_DIR
+        """Geth data directory for op-geth node."""
+
+        self.chaindata_dir = f"{self.data_dir}/geth/chaindata"
+        """Directory storing chain data."""
+
+        self.chain_id = 42069
+        """Chain ID of the local L2."""
+
+        # For the following values, allow environment override for now, to follow the original.
+        # In due time, remove that as we provide our own way to customize.
+
+        self.verbosity = os.environ.get("GETH_VERBOSITY", 3)
+        """Geth verbosity level (from 0 to 5, see geth --help)."""
+
+        self.rpc_port = os.environ.get("RPC_PORT", 9545)
+        """Port to use for the http-based JSON-RPC server."""
+
+        self.ws_port = os.environ.get("WS_PORT", 9546)
+        """Port to use for the WebSocket-based JSON_RPC server."""
+
```

Comment:
We probably should get rid of the env override. We don't use them and we haven't created them for the other config values.